e2e/utils/pages/homepage.page.ts

```diff
@@ -55,10 +57,12 @@ interface WaitForLocatorOptions {
 abstract class Homepage {
 	readonly #page: Page;
 	readonly #viewportSize?: ViewportSize;
+	readonly promotionCarousel: PromotionCarousel;
 
 	protected constructor({ page, viewportSize }: HomepageParams) {
 		this.#page = page;
 		this.#viewportSize = viewportSize;
+		this.promotionCarousel = new PromotionCarousel(page);
```

Comment:
do we want to initiate it here? shouldn't it be inside some sort of awaiting function? so that we can avoid fails on initiation, but we check for the existence of the carousel too

Reply:
as discussed offline, all test should be agnostic to the carousel, since it is tested elsewhere. If we don't let all tests for Homepage fail on initiation, we can pinpoint better what is failing

Reply:
done